app/models/gobierto_data/dataset.rb

```diff
@@ -95,6 +95,17 @@ def columns_stats
       end
     end
 
+    def rows_count
+      cache_service.fetch("#{cache_key_with_version}/rows_count") do
+        count_result = ::GobiertoData::Connection.execute_query(
+          site,
+          Arel.sql("SELECT COUNT(1) FROM #{table_name} LIMIT 1"),
+          include_draft: true
```

Comment:
Would it makes sense to try what mentions here: https://arctype.com/blog/postgres-count/ ?

I mean the ```
SELECT
    reltuples::bigint AS estimate
FROM
    pg_class
WHERE
    oid = 'public.sample_data' ::regclass;
```

Reply:
Because we just want an estimation, right?